resources/js/views/meetings/Index.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <b-container class="mt-3 mb-5">
+    <b-row class="mb-3">
+      <b-col>
+        <h2>
+          {{ $t('meetings.currentlyRunning') }}
+        </h2>
+      </b-col>
+      <b-col sm='12' md='3'>
+        <b-input-group>
+          <b-form-input @change="$root.$emit('bv::refresh::table', 'meetings-table')" :disabled="isBusy || loadingError" ref="search" :placeholder="$t('app.search')" v-model="filter"></b-form-input>
+          <b-input-group-append>
+            <b-button @click="$root.$emit('bv::refresh::table', 'meetings-table')" :disabled="isBusy || loadingError" variant="success"><b-icon icon="search"></b-icon></b-button>
+          </b-input-group-append>
+        </b-input-group>
+      </b-col>
+    </b-row>
+    <hr>
+
+    <b-overlay  :show="loadingError" >
+
+      <template #overlay>
+        <div class="text-center my-2">
+          <b-spinner v-if="isBusy" ></b-spinner>
+          <b-button
+            v-else
+            @click="$root.$emit('bv::refresh::table', 'meetings-table')"
+          >
+            <b-icon-arrow-clockwise></b-icon-arrow-clockwise> {{ $t('app.reload') }}
+          </b-button>
+        </div>
+      </template>
+
+      <b-table
+      fixed
+      hover
+      stacked='lg'
+      show-empty
+      :busy.sync='isBusy'
+      :fields='tableFields'
+      :items='fetchMeetings'
+      id='meetings-table'
+      ref='meetings'
+      :current-page='currentPage'>
+
+      <template v-slot:empty>
+        <i>{{ filtered ? $t('meetings.nodataFiltered') : $t('meetings.nodata') }}</i>
```

Comment:
For filtered there is an extra slot, named `emptyfiltered`

Reply:
that is true, but as we don't use a client side filter function we don't know if there are not results or just no results for the search query. So we need this workaround to show the user a different message if a search query is present.

> Content to display when no items are present in the filtered `items` array. Optionally scoped

Reply:
The problem is that you just not set the filter property on the table. In that case the @change-listener in the filter input field wouldn't be necessary, because the function for getting the data would be triggered automatically!

Reply:
This is already used in the users index table for example!

Reply:
Fixed. I did not use the debounce feature, but lazy to send less requests to the server